user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -46,6 +46,8 @@ Others
 - ``CITestStreamFilter::$buffer = ''`` no longer causes the filter to be registered to listen for streams. Now there
   is a ``CITestStreamFilter::registration()`` method for this. See :ref:`upgrade-430-stream-filter` for details.
 - ``Forge::addUniqueKey()`` now applies ``_u`` suffix to the constraint name. This was done to ensure a unique index name.
+- The data structure returned by ``BaseConnection::getForeignKeyData()`` has been changed.
+- The naming of foreign keys in OCI8 has changed. The previous suffix of ``_fk`` has changed to ``_foreign``.
```

Comment:
In the current code in this PR I see quite different behavior. This description should be updated.

Reply:
Ya, thought I had changed that.. fixed it now.